subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/ivyresolve/parser/GradlePomModuleDescriptorParser.java

```diff
@@ -202,6 +202,16 @@ private PomReader parseImportedPom(DescriptorParseContext parseContext, PomDepen
     private PomReader parseOtherPom(DescriptorParseContext parseContext, ModuleComponentIdentifier parentId) throws IOException, SAXException {
```

Comment:
As shown in http://maven.apache.org/ref/3-LATEST/maven-model-builder/index.html (thanks @fbiville), there seem to be too many cases for a single PR, what do you think ?